com.eclipsesource.modelserver.emf/src/main/java/com/eclipsesource/modelserver/emf/configuration/ServerConfiguration.java

```diff
@@ -42,4 +44,21 @@ public void setServerPort(int serverPort) {
 		this.serverPort = serverPort;
 	}
 
+	public static boolean isValidWorkspaceRoot(String fileUrl) {
+		if (fileUrl != null) {
+			File file = new File(toFilePath(fileUrl));
+			return file.exists();
+		}
+		return false;
+	}
+
+	public static boolean isValidPort(Integer port) {
+		return port >= 0 || port <= 65535;
+
+	}
+
+	private static String toFilePath(String url) {
+		return url.replace("file://", "");
```

Comment:
I think we could make this a bit more robust by creating an actual `URL` and then call `getPath` on it.

Reply:
True that. I pushed a change for that. Could you have a quick look?